fabric-mining-level-api-v1/src/testmod/resources/assets/fabric-mining-level-api-v1-testmod/lang/en_us.json

```diff
@@ -0,0 +1,8 @@
+{
+  "item.minecraft.shears": "Iron Shears",
```

Comment:
Not really needed in a testmod

Reply:
eh, neither are things like textures, but i figured 'why not'. after all, it doesn't hurt